dali/dfu/dfuutil.cpp

```diff
@@ -1296,6 +1296,24 @@ class CDFUhelper: implements IDFUhelper, public CInterface
         }
     }
 
+    MemoryBuffer &getFileHistory(const char *lfn,MemoryBuffer &out, IUserDescriptor *user)
```

Comment:
Something like "seraizlieFileHistory" would be a more conventional name for this type of method.